.github/workflows/plugins.yml

```diff
@@ -1,49 +1,70 @@
 name: plugins
 on:
+# Run CI against all pushes (direct commits) and Pull Requests
   push:
-    paths:
-      - 'plugins/**'
-      - 'tests/**'
-      - 'scripts/**'
-      - '.github/workflows/plugins.yml'
-      - 'docker-compose.yml'
+#    paths:
```

Comment:
What about these? We preciously added them so integration tests are not run everytime a change is made in the roles. But we may change the logic to ignore_paths

Reply:
How about keeping them just for PRs?

Reply:
@D3DeFi Good call on `ignore_paths`. I hope at some point we will have `ansible-test` validate galaxy.yml etc, so this feels safer.

Maybe in the future we might see some Molecule testing for these `roles` https://github.com/ansible-collections/community.kubernetes/blob/master/.github/workflows/ci.yml#L67-L99 ...